airbyte-ci/connectors/connector_ops/tests/test_required_reviewer_checks.py

```diff
@@ -37,7 +37,7 @@ def not_tracked_change_expected_team(tmp_path, pokeapi_metadata_path):
     backup_path = tmp_path / "non_strategic_acceptance_test_config.backup"
     shutil.copyfile(pokeapi_metadata_path, backup_path)
     with open(pokeapi_metadata_path, "a") as metadata_file:
-        metadata_file.write("not_tracked")
+        metadata_file.write("\nnot_tracked: true\n")
```

Comment:
This was throwing a YAML SafeLoader error, needed to be formatted as a `k:v` pair